api/all.yaml

```diff
@@ -2097,6 +2097,21 @@ components:
           type: integer
           format: int64
       type: object
+    UserResource:
```

Comment:
Could this be in `control.yaml`? I don't think there's anything else in `all.yaml` that's referencing it, is there?

Reply:
Server need auto generated models from this file. Moving to control.yaml is a bit more complicated and risky because we will have to generate from 2 separate files.

Reply:
oh nvm you are right I can probably move this, server is using the actual scim class.